policies/travel-policy.md

```diff
@@ -165,51 +197,54 @@ OpenSSL will reimburse costs incurred for:
 -   Room type should be a Standard Hotel Room. The Traveller may upgrade
     the room at their own expense.
 -   OpenSSL will fund 1 day's accommodation prior to the required travel
-    and 1 day post the required travel time. Refer Travel Class Table
-    for when additional pre-days can be reimbursed.
+    and 1 day post the required travel time.
 -   Use of the minibar, laundry services or other hotel facilities
     (excluding those required for Business purposes e.g. meeting rooms)
     are at the Traveller's own expense.
 
 ### Meals
 
-The amount listed in the **Meal Entitlement Table** will be the maximum
+The amount listed in the [Meal Entitlement Table] will be the maximum
 amount that can be claimed for each day, it is a per diem amount i.e.
 the allowance is inclusive of all meals that occurred on the relevant
 Day.
 
-Only the OMC can amend the Meal Entitlement Table.
+Only the OMC can amend the [Meal Entitlement Table].
 
 OpenSSL will only pay for meals related to the Traveller and will not
 pay for meals related to other parties e.g. guests, partners, friends
-except where the other party is also travelling on behalf of OpenSSL in
-these circumstances OpenSSL will pay the meal allowance applicable to
-each Traveller for that day.
+except where the other party is also travelling on behalf of OpenSSL.
+In these circumstances OpenSSL will pay the meal allowance applicable
+to each Traveller for that day.
```

Comment:
Replace `will pay the meal allowance` with `will reimburse meals`

Reply:
Changed